ts/features/itwallet/analytics/index.ts

```diff
@@ -0,0 +1,476 @@
+import { mixpanelTrack } from "../../../mixpanel";
+import { updateMixpanelProfileProperties } from "../../../mixpanelConfig/profileProperties";
+import { GlobalState } from "../../../store/reducers/types";
+import { buildEventProperties } from "../../../utils/analytics";
+import { CredentialType } from "../common/utils/itwMocksUtils";
+import { IdentificationContext } from "../machine/eid/context";
+import {
+  ITW_ACTIONS_EVENTS,
+  ITW_CONFIRM_EVENTS,
+  ITW_ERRORS_EVENTS,
+  ITW_EXIT_EVENTS,
+  ITW_SCREENVIEW_EVENTS,
+  ITW_TECH_EVENTS
+} from "./enum";
+
+export type KoState = {
+  reason: string;
+  cta_category: "custom_1" | "custom_2";
+  cta_id: string;
+};
+
+const mixPanelCredentials = ["ITW_ID", "ITW_PG", "ITW_CED", "ITW_TS"] as const;
+
+type MixPanelCredential = (typeof mixPanelCredentials)[number];
+
+export type OtherMixPanelCredential = "welfare" | "payment_method" | "CGN";
+type NewCredential = MixPanelCredential | OtherMixPanelCredential;
+
+export const CREDENTIALS_MAP: Record<CredentialType, MixPanelCredential> = {
+  PersonIdentificationData: "ITW_ID",
+  MDL: "ITW_PG",
+  EuropeanDisabilityCard: "ITW_CED",
+  EuropeanHealthInsuranceCard: "ITW_TS"
+};
```

Comment:
`CredentialType` is a mock value that will be discontinued in the future, use `string` instead

```suggestion
export const CREDENTIALS_MAP: Record<string, MixPanelCredential> = {
  PersonIdentificationData: "ITW_ID",
  MDL: "ITW_PG",
  EuropeanDisabilityCard: "ITW_CED",
  EuropeanHealthInsuranceCard: "ITW_TS"
};
```